src/scripts/main.js

```diff
@@ -354,7 +354,48 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const newRow = document.createElement('tr');
+  table.append(newRow);
+
+  function sexChecking(value) {
+    if (value === 'm') {
+      return 'Male';
+    }
+
+    if (value === 'f') {
+      return 'Female';
+    }
+  }
```

Comment:
The `sexChecking` function could be simplified using a ternary operator: `return value === 'm' ? 'Male' : 'Female';`. This would make the code more concise and readable.